### DISCUSSION TITLE
Search button visible

### DISCUSSION THREAD

**Discussion by actuariat** [quarto-dev/quarto-cli#7934] [Feature Requests]
### Description

For mobile users it is quite impossible to search inside quarto presentations. Can you set a visible button to fix that?

**Comment by mcanouil** [comment]
What do you mean?

Fore Reveal.js, you need to create or use a plugin.
Note that searching within slides of mobile seems to be really a niche.

**Reply by actuariat** [reply]
On a computer one presses command+shift+F to search inside a quarto presentation. I am asking for the same box to be visible when on mobile device (iPad or similar). On a tablet it is impossible to search.

**Reply by mcanouil** [reply]
This is Reveal.js framework.
Note that slides are not really intended for mobile use in the first place.

You gonna have to look for a Reveal.js plugin that does that.

The "RevealSearch" plugin is from Reveal.js. Quarto is only using it, see <https://revealjs.com/plugins/>.
I suggest to make your request in Reveal.js repository: <https://github.com/hakimel/reveal.js/issues> (if not already there).

**Reply by actuariat** [reply]
ok, thank you. I thought, since the plugin exists, that the problem would only be to make it accessible. I post is on reveal.js

**Reply by mcanouil** [reply]
Yes, but it means to modify the plugin itself which is not developed by Quarto.
